CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+1. modifying the values of the protocol parameters (including initiating "hard forks"); and
+2. transferring Ada out of the reserves and the treasury (and also moving Ada between the reserves and the treasury).
+
+In the current scheme, governance actions are initiated by special transactions that require `Quorum-Many` authorizations from the governance keys (5 out of 7 on the Cardano mainnet)[^1].
+Fields in the transaction body provide details of the action to be carried out: whether changing protocol parameter changes or initiating funds transfers. Each transaction can trigger precisely one kind of action, but a single action can make multiple changes.
+
+- Protocol parameter updates use [transaction field nº6](https://github.com/input-output-hk/cardano-ledger/blob/8884d921c8c3c6e216a659fca46caf729282058b/eras/babbage/test-suite/cddl-files/babbage.cddl#L56) of the transaction body.
+- Movements of the treasury and the reserves use [Move Instantaneous Rewards (abbrev. MIR) certificates](https://github.com/input-output-hk/cardano-ledger/blob/8884d921c8c3c6e216a659fca46caf729282058b/eras/babbage/test-suite/cddl-files/babbage.cddl#L180).
+
+Successful governance actions are applied on an epoch boundary (they are **enacted**).
+
+One of the protocol parameters is sufficiently significant to merit special attention: changing the major protocol version enables Cardano to enact controlled hard forks. This type of update, therefore, has a special status among the possible protocol parameter updates.
+
+### Shortcomings of the Shelley Governance Design
+
+The current design was intended to provide a simple, transitional approach to governance.
+This proposal aims to address a number of shortcomings with that design.
+
+1. It gives no room for active on-chain participation of Ada holders. While changes to the protocol are usually the results of discussions with selected community actors, the process is driven mainly by the founding entities.
+Ensuring everyone can voice their concern is cumbersome and can be perceived as arbitrary at times.
+
+2. Movements from the treasury can be hard to track and constitute a critical and sensitive topic.
+It is important to have more transparency and more layers of control over these movements.
+
+3. While they need to be treated specially by SPOs, hard forks are not differentiated from other protocol parameter changes.
+
+4. Finally, while there's currently a somewhat common vision for _Cardano_ that is shared by its founding entities and many community members, there is no clearly defined document where these guiding principles are recorded. It makes sense to leverage the Cardano blockchain to record the ethos of the project itself in an immutable fashion, as the formal Cardano Constitution.
+
+## Specification
+
++ [The Cardano Constitution](#the-constitution)
++ [The Constitutional Committee](#the-constitutional-committee)
```

Comment:
The constitutional committee should be remained to the "Central Committee". If the Central Committee will have veto power over proposals that have been approved by dRep/SPOs and within the scope of the [Cardano Constitution] there is nothing constitutional about it. Similar in light to the DPRK. Is North Korea really democratic if its in their name?

Reply:
Anything but direct democracy inherently involves some level of centralization. This is why I don't like dreps - their whole purpose is to concentrate power, exactly the opposite of what a decentralized system should do imho.